sdk/containerregistry/azure-containerregistry/azure/containerregistry/aio/_async_container_registry_client.py

```diff
@@ -864,3 +864,24 @@ async def delete_manifest(self, repository: str, tag_or_digest: str, **kwargs) -
             tag_or_digest = await self._get_digest_from_tag(repository, tag_or_digest)
 
         await self._client.container_registry.delete_manifest(repository, tag_or_digest, **kwargs)
+
+    @distributed_trace_async
+    async def delete_blob(self, repository: str, digest: str, **kwargs) -> None:
+        """Delete a blob. If the blob cannot be found or a response status code of
+        404 is returned an error will not be raised.
```

Comment:
It says here that an error will not be raised in the case of a 404, but I don't see any handling here of a 404. The generated code will raise on a 404. So does the service never return a 404? Or should we be handling it here?

Reply:
Service does return a 404 error `ResourceNotFoundError` when blob to delete is not found, we are not handling the error here. The docstring here is not accurate.